Assets/Python/Consts.py

```diff
@@ -1707,22 +1682,18 @@
 #leaders
 
 iNumLeaders = 114
-(iLeaderBarbarian, iNativeLeader, iIndependentLeader, iAlexander, iAsoka, iAugustus, iBismarck, iBoudica, iBrennus, iCatherine, iCharlemagne, iChurchill,

```

Comment:
I prefer the shortened names because they save time typing. The other corrections (where the name actually changed) are welcome though.